pkg/transform/routes.go

```diff
@@ -49,6 +49,8 @@ type RoutesConfig struct {
 	Patterns       []string   `yaml:"patterns"`
 	IgnorePatterns []string   `yaml:"ignored_patterns"`
 	IgnoredEvents  IgnoreMode `yaml:"ignore_mode"`
+	// Character that will be used to replace route segments
+	WildcardChar string `yaml:"wildcard_char,omitempty"`
```

Comment:
I'm wondering if it's possible to use 'char' here, I've never tried it. If not then string that's validates works for me.

Reply:
@grcevski hmm. I will give that a try. I attempted rune before but the yaml unmarshaller does not like that. It seems like it is not possible to unmarshal directly into the rune type.

Reply:
I seemingly misunderstood something - I thought there was some 'char' type I did not know about but apparently for Go that is just 'rune'. I confirmed it does not work to unmarshal directly into rune here so we need to keep it to string.